x-pack/plugins/security_solution/public/resolver/view/panels/panel_content_related_list.tsx

```diff
@@ -34,16 +36,56 @@ interface MatchingEventEntry {
   setQueryParams: () => void;
 }
 
+const StyledRelatedLimitWarning = styled(RelatedEventLimitWarning)`
+  flex-flow: row wrap;
+  display: block;
+  align-items: baseline;
+  margin-top: 1em;
+
+  & .euiCallOutHeader {
+    display: inline;
+    margin-right: 0.25em;
+  }
+
+  & .euiText {
+    display: inline;
+  }
+
+  & .euiText p {
+    display: inline;
+  }
+`;
+
 const DisplayList = memo(function DisplayList({
   crumbs,
   matchingEventEntries,
+  eventType,
+  processEntityId,
+  aggregateCountForEventType,
 }: {
   crumbs: Array<{ text: string | JSX.Element; onClick: () => void }>;
   matchingEventEntries: MatchingEventEntry[];
+  eventType: string;
+  processEntityId: string;
+  aggregateCountForEventType: number;
 }) {
+  const relatedEventResponsesById = useSelector(selectors.relatedEventsByEntityId);
+  const responseForThisNode = relatedEventResponsesById.get(processEntityId);
+  const shouldShowLimitWarning =

```

Comment:
please include a unit test for `shouldShowLimitWarning`.

`aggregateCountForEventType` has logic to define a default below: 
![image](https://user-images.githubusercontent.com/35559/86133173-f7f38500-bab5-11ea-93c5-a5ffdb047fd7.png)

Please make sure that default logic is represented in the unit test.